### PR TITLE
[APHL-652] add requireNonExperimental parameter to $release

### PR DESCRIPTION
**Pull request by TahaAttari**

### CODE REVIEW

File: input/fsh/release-operation.fsh [hunk 1]
```diff
@@ -66,4 +66,13 @@ Usage: #definition
 * parameter[=].min = 0
 * parameter[=].max = "1"
 * parameter[=].documentation = "Indicates whether or not the terminology server from which a value set was originally downloaded should be checked for the latest version. The terminolgy server of origin is tracked via the authoritativeSource extension (https://hl7.org/fhir/extension-valueset-authoritativesource.html) on the value set. If this flag is set to false or the value set does not have an authoritativeSource specified, then the check should be constrained to the local system/cache. (default = false)"
-* parameter[=].type = #boolean
\ No newline at end of file
+* parameter[=].type = #boolean
```
Comment (sliver007) [inline input/fsh/release-operation.fsh:69]: This should be a code, right?

File: input/fsh/release-operation.fsh [hunk 2]
```diff
@@ -66,4 +66,13 @@ Usage: #definition
...
+
+* parameter[+].name = #requireNonExperimental
```
Comment (sliver007) [inline input/fsh/release-operation.fsh]: I think we settled on calling this "experimentalBehavior".

File: input/fsh/release-operation.fsh [hunk 3]
```diff
@@ -66,4 +66,13 @@ Usage: #definition
...
+* parameter[=].use = #in
+* parameter[=].min = 0
+* parameter[=].max = "1"
+* parameter[=].documentation = "Indicates whether or not the repository should throw an error or warning if a specification which is not Experimental references Experimental components"
```
Comment (sliver007) [inline input/fsh/release-operation.fsh]: The third option here is when behavior: none - in that case the Experimental element isn't validated at all and so maybe in the description here we should mention that scenario?

File: input/pages/artifact-repository-service.md [hunk 4]
```diff
@@ -225,6 +225,7 @@ The following parameters SHOULD be supported for the operation:
     * **check**: Edge Case: If a library or value set specifies a different version, an error is returned instead of the package.
     * **force**: Edge Case: This parameter overrides any specified version in the library and value sets (and any it depends on). Note that this has obvious safety issues, in that it may result in a value set expansion giving a different list of codes that is both wrong and unsafe, and implementers should only use this capability reluctantly. It primarily exists to deal with situations where specifications have fallen into decay as time passes. If the value is override, the version used SHALL explicitly be represented in the expansion parameters
 * **latestFromTxServer**: Whether or not the repository should search the remote source when updating references.
+* **requireNonExperimental**: Whether or not the repository should throw an error if the artifact being released is not Experimental and references Experimental components
```
Comment (sliver007) [inline input/pages/artifact-repository-service.md]: Same as comments above - name: experimenatBehavior and also include description of the 'none' behavior.

File: input/pages/operations.md [hunk 5]
```diff
@@ -136,6 +136,7 @@ Options for federated client specification:
     * resource
     * versionBehavior
     * latestFromTxServer
+    * requireNonExperimental
```
Comment (sliver007) [inline input/pages/operations.md]: Name: experimentalBehavior

**Review by sliver007 — CHANGES_REQUESTED** [review]
A few small changes requested.

File: input/fsh/release-operation.fsh (hunk 1 above)
Comment (TahaAttari) [reply input/fsh/release-operation.fsh:69]: this is for `latestFromTxServer`, I think that's still a boolean 👍🏾

File: input/pages/artifact-repository-service.md (hunk 4 above)
Comment (TahaAttari) [reply input/pages/artifact-repository-service.md]: updated 👍🏾

File: input/fsh/release-operation.fsh (hunk 3 above)
Comment (TahaAttari) [reply input/fsh/release-operation.fsh]: updated 👍🏾

File: input/fsh/release-operation.fsh (hunk 2 above)
Comment (TahaAttari) [reply input/fsh/release-operation.fsh]: updated 👍🏾

**Review by sliver007 — APPROVED** [review]
Looks good. Thank you!

**Pull request merged**